cypress/integration/AppAnalytics/BasicSearchResultsPageLoad.feature

```diff
@@ -1,32 +1,161 @@
+			| page.additionalDetails.numberResults     | (int)11189                                         |
+			| page.additionalDetails.searchKeyword     | tumor                                              |
+			| page.additionalDetails.itemsPerPage      | (int)20                                            |
+			| page.additionalDetails.pageNum           | (int)2                                             |
+			| page.additionalDetails.showingBestBets   | (bool)false                                        |
+			| page.additionalDetails.showingDefinition | (bool)false                                        |
+
+	Scenario: Page Load Analytics fires when a user searches for a keyword with no results, Best Bet and Definition not displayed
+		Given  "title" is set to "NCI Search Results"
+		And "language" is set to "en"
+		And "baseHost" is set to "http://localhost:3000"
+		And "canonicalHost" is set to "https://www.cancer.gov"
+		And "siteName" is set to "National Cancer Institute"
+		And "channel" is set to "Search"
+		And "analyticsContentGroup" is set to "Global Search"
+		And "analyticsPublishedDate" is set to "02/02/2011"
+		When the user navigates to "/?swKeyword=achoo"
+		And the page displays "0 results found for: achoo"
+		And browser waits
+		Then there should be an analytics event with the following details
+			| key                                      | value                                          |
+			| type                                     | PageLoad                                       |
+			| event                                    | SiteWideSearchApp:Load:Results                 |
+			| page.name                                | www.cancer.gov/?swKeyword=achoo                |
+			| page.title                               | NCI Search Results                             |
+			| page.metaTitle                           | NCI Search Results - National Cancer Institute |
+			| page.language                            | english                                        |
+			| page.type                                | nciAppModulePage                               |
+			| page.channel                             | Search                                         |
+			| page.contentGroup                        | Global Search                                  |
+			| page.publishedDate                       | 02/02/2011                                     |
+			| page.additionalDetails.searchKeyword     | achoo                                          |
+			| page.additionalDetails.numberResults     | (int)0                                         |
+			| page.additionalDetails.itemsPerPage      | (int)20                                        |
+			| page.additionalDetails.pageNum           | (int)1                                         |
+			| page.additionalDetails.showingBestBets   | (bool)false                                    |
+			| page.additionalDetails.showingDefinition | (bool)false                                    |
```

Comment:
You already have this test in NoResultsFoundAnalytics feature - since it is the same data, one of them should be removed and save us some build time:)

Reply:
Quite true. Removed.